### ISSUE TITLE
Provide a simpler and more realistic example script that uses async

### ISSUE THREAD

**Issue by ctbur** [ctbur/async#2]

**Comment by davidjeddy** [comment]
Would a multi-file download (wget/curl) be acceptable?

**Comment by ctbur** [comment]
Yes, that's clearly better than the existing example. Maybe downloading and decompressing several tar files.

**Comment by UnixJunkie** [comment]
In a competing project (https://github.com/UnixJunkie/pardi), here is my demo script:

Compress a file in parallel using 1MB chunks:
```
pardi -d b:1048576 -m s -i <YOUR_BIG_FILE> -o <YOUR_BIG_FILE>.gz -w 'xz -c -9 %IN > %OUT'
```

**Comment by UnixJunkie** [comment]
I have another competing project also, this one older:
https://github.com/UnixJunkie/PAR

It's fun that several people are not satisfied with GNU parallel.
